docs/articles/pkgdown/quickstart.Rmd

```diff
@@ -134,7 +134,7 @@ summary(corpus(mytf7), 5)
 
 ### Corpus principles
 
-A corpus is designed to be a "library" of original documents that have been converted to plain, UTF-8 encoded text, and stored along with meta-data at the corpus level and at the document-level.  We have a special name for document-level meta-data: *docvars*.  These are variables or features that describe attributes of each document.
+A corpus is designed to be a "library" of original documents that have been converted to plain, UTF-8 encoded text, and stored along with meta-data at the corpus level and at the document-level.  We have a special name for document-level meta-data: `docvars()`.  These are variables or features that describe attributes of each document.
```

Comment:
Let's keep "_docvars_" (italicised) since it refers here to the name of this element, not to the function.